R/tax_range_time.R

```diff
@@ -0,0 +1,178 @@
+#' Calculate the temporal range of fossil taxa
+#'
+#' A function to calculate the temporal range of fossil taxa.
+#'
+#' @param occdf \code{dataframe}. A dataframe of fossil occurrences. The
+#' dataframe should contain at least three columns: names of taxa (`character`)
+#' and the maximum (`numeric`) and minimum (`numeric`) age which constrain the
+#' age range (in millions of years) of the fossil occurrence. If the age range
+#' columns are provided as (`character`) values (i.e., stage names), the
+#' function will attempt to extract numeric age data from the
+#' Geological Timescale by linking stage names.
+#' @param name \code{character}. The name of the column you wish to be treated
+#' as the input names (e.g., "species" or "genus").
+#' @param min_ma \code{character}. The name of the column you wish to be treated
+#' as the input minimum age range (e.g., "min_ma").
+#' @param max_ma \code{character}. The name of the column you wish to be treated
+#' as the input maximum age range (e.g., "max_ma").
+#' @param scale \code{character}. Specify the desired geological timescale to
+#' be used "GTS2020" or "GTS2012". This argument is only useful if the supplied
+#' "max_ma" and "min_ma" columns are of class \code{character}.
+#' "GTS2020" is the default.
```

Comment:
Could link to look_up() here?

Reply:
Added, this will likely show up as an error until both branches are merged though!